### DISCUSSION TITLE
Approval levels for CIP process changes

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#10] [Medium CIPs]
Any changes to CIP process and approval mechanism should follow CIP process.

Currently the process only suggests "major changes" to CIP process are Large changes.

It would be worth clarifying that.

Below a list to start off the discussion.

Any additions / views? 

small: 
- process clarification

medium:
- process changes

large:
- who approves what levels
- what levels of changes are there
- level definitions

**Comment by addanus** [comment]
This is kind of meta, but definitely worth discussion/approval.

My response: I second the proposal

**Reply by tijno** [reply]
thx @addanus !

Also looks like @chafreaky @hpaulson @N0rdian and @zopel7 are happy with this.

@maebeam whats the next step on this? I think it needs core dev approval - so would be good to get some feedback from you as the original author of CIP.

**Reply by maebeam** [reply]
I approve

**Reply by tijno** [reply]
Ok if I turn this straight into a PR @maebeam ? I think its a process clarification rather then a change.

**Reply by maebeam** [reply]
Yes this feels like a clarification to me

**Reply by tijno** [reply]
Ok done @maebeam  #50
